lib/Auth/Source/PrivacyideaAuthSource.php

```diff
@@ -119,10 +119,16 @@ public function authenticate(&$state)
         $state['privacyidea:privacyidea:ui']['webAuthnSignRequest'] = "";
         $state['privacyidea:privacyidea:ui']['u2fSignRequest'] = "";
         $state['privacyidea:privacyidea:ui']['mode'] = "otp";
-        $state['privacyidea:privacyidea:ui']['otpFieldHint'] = @$this->authSourceConfig['otpFieldHint'] ?: "";
-        $state['privacyidea:privacyidea:ui']['passFieldHint'] = @$this->authSourceConfig['passFieldHint'] ?: "";
         $state['privacyidea:privacyidea:ui']['otpExtra'] = @$this->authSourceConfig['otpExtra'] ?: false;
         $state['privacyidea:privacyidea:ui']['loadCounter'] = "1";
+        if(!empty($this->authSourceConfig['otpFieldHint']))
```

Comment:
We should probably use `isset` here

Reply:
Fixed: [1257aa3](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/162/commits/1257aa31d567994f0487371fe92a7c3884c02462)